app/models/transformation_mapping/vm_migration_validator.rb

```diff
@@ -10,9 +10,10 @@ class TransformationMapping::VmMigrationValidator
   VM_NOT_EXIST = "not_exist".freeze
   VM_VALID = "ok".freeze
 
-  def initialize(mapping, vm_list = nil)
+  def initialize(mapping, vm_list = nil, service_template_id = nil)
     @mapping = mapping
     @vm_list = vm_list
+    @service_template_id = service_template_id.to_i
```

Comment:
This can hit a null pointer if the param is `nil`.  Probably just need to protect against this, or decide that the `service_template_id` param can't be `nil`.

Reply:
Ok, I addressed that with -
```rb
@service_template_id = service_template_id.try(:to_i)
```

Reply:
Interestingly enough, `nil.to_i` returns 0 :)